server/cluster/cluster.go

```diff
@@ -554,10 +556,11 @@ func (c *RaftCluster) HandleStoreHeartbeat(stats *pdpb.StoreStats) error {
 		}
 		peerInfo := core.NewPeerInfo(peer, 0, 0,
 			peerStat.GetReadBytes(), peerStat.GetReadKeys(), interval)
-		item := statistics.NewFlowItem(peerInfo, region, nil)
+		item := statistics.NewPeerInfoItem(peerInfo, region)
 		c.hotStat.CheckReadAsync(item)
 	}
-
+	coldItem := statistics.NewColdItem(storeID, regionIDs, interval)
```

Comment:
`coldItem` is confusing, we need a more fit name.

Reply:
updated.